### ISSUE TITLE
Correctly expose the visual hierarchy of accessibility nodes

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#1672]
From `FlutterSemanticsToAndroidAccessibilityBridge.java`:

``` java
        // TODO(ianh): use setTraversalBefore/setTraversalAfter to set                                                                                                                                                                 
        // the relative order of the views. For each set of siblings,                                                                                                                                                                  
        // the views should be ordered top-to-bottom, tie-breaking                                                                                                                                                                     
        // left-to-right (right-to-left in rtl environments), height,                                                                                                                                                                  
        // width, and finally by list order.                                                                                                                                                                                           
```

**Comment by Hixie** [comment]
This shouldn't be so hard on Android, because we already have the global coordinates for every box so it's "just" a matter of sorting the list and calling the API.

**Comment by goderbauer** [comment]
This has been fixed.
